lib/src/rules/prefer_contains.dart

```diff
@@ -0,0 +1,207 @@
+// Copyright (c) 2016, the Dart project authors.  Please see the AUTHORS file
+
```

Comment:
Remove blank line, also consider "Sort members in dart file" (available in the Code menu in intellij if the dart plugin is installed). I don't quite like the implemented order there, but it avoid discussions, is easy and fast to do.

I think there is no requirement/policy around this, so is up to you.